tests/test_app.py

```diff
@@ -85,6 +87,7 @@ def test_app(
     assert len(db_session.query(ForecastValueSevenDaysSQL).all()) == expected_forecast_results * 16
 
 
+@pytest.mark.skip(reason="The day ahead model has not yet been re-trained with data-sampler")
```

Comment:
oooh, not sure how I feel about this. 

This is bad incase we need a new version of DA suddently

Reply:
What kind of scenario are you thinking of? One where we need to retrain?

Reply:
nah, just putting it back, so it runs on legacy ocf_datapipes